src/plugins/omake/OMakeEquip.ml

```diff
@@ -331,6 +331,20 @@ let get_lib_deps ?(transitive=false) ?(filter = fun _ -> true) pkg bs =
        bs.bs_build_depends
     )
 
+let get_sorted_trans_lib_deps pkg bs =
```

Comment:
Can this really happen? This function is mainly using OASISGraph.topological_sort, which is only returning a node when it is first visited in a depth-first traversal of the graph. Unless I'm overlooking something, the list of nodes will only contain at most one instance of a node.